src/main/component/Editor.svelte

```diff
@@ -12,6 +12,10 @@
     return prefix + '_' + Math.floor(Math.random() * 1000000000) + String(Date.now());
   };
 
+  const isDisabledOptionSupported = (editor: TinyMCEEditor): boolean => {
```

Comment:
I think this check needs to be different after talking to @michalnieruchalski-tiugo since we want to feed readonly or disabled in at init time as well and then we don't have the editor instance.

Reply:
If they use disabled=true on a old editor version then it wouldn't be disabled since that option didn't exist prior to 7.6.

Reply:
@spocke, @michalnieruchalski-tiugo  I fixed it, with a check in the setup, let me know what do you think about it